vavr/src/main/java/org/jdbi/v3/vavr/VavrCollectorFactory.java

```diff
@@ -70,14 +78,19 @@
                 Tuple.of(List.class, List.collector()),
                 Tuple.of(Stream.class, Stream.collector()),
                 Tuple.of(Queue.class, Queue.collector()),
+                Tuple.of(PriorityQueue.class, PriorityQueue.collector()),
                 // Sets
                 Tuple.of(HashSet.class, HashSet.collector()),
                 Tuple.of(LinkedHashSet.class, LinkedHashSet.collector()),
                 Tuple.of(TreeSet.class, TreeSet.collector()),
                 // Maps
                 Tuple.of(HashMap.class, HashMap.collector()),
                 Tuple.of(LinkedHashMap.class, LinkedHashMap.collector()),
-                Tuple.of(TreeMap.class, TreeMap.collector())
+                Tuple.of(TreeMap.class, TreeMap.collector()),
+                // Multimaps
+                Tuple.of(HashMultimap.class, HashMultimap.withSeq().collector()),
```

Comment:
just wanted to mention: the multimap backed by seq seems like a sensible default. but somehow it might be nice to have that configuraable. currently `CollectorFactory` has no injection of configuration.

I think this is fine, since you can explicitly request collection with Multimap backed by Set or whatever (via `collect` method) if needed just wanted to add that thought here

Reply:
We can start adding configuration if necessary, although usually my preference would simply be that the application registers their own mappers after all the defaults get pulled in - last registration wins